Packet++/header/LdapLayer.h

```diff
@@ -1050,3 +1254,17 @@ inline std::ostream& operator<<(std::ostream& os, const pcpp::LdapAttribute& att
 	os << "{" << attr.type << ", {" << valuesStream << "}}";
 	return os;
 }
+
+inline std::ostream& operator<<(std::ostream& os, const pcpp::LdapBindRequestLayer::SaslAuthentication& saslAuthentication)
+{
+	std::string valuesStream;
```

Comment:
```
const char* separator = "";
for (const auto& value : saslAuthentication.credentials)
{
	os << "0x" << value;
	separator = ", ";
}
os << "{" << saslAuthentication.mechanism << ", {" << valuesStream << "}}";
return os;
```

Reply:
I like this suggestion! Implemented for both `SaslAuthentication` and `LdapAttribute` in this commit: https://github.com/seladb/PcapPlusPlus/pull/1448/commits/57fa2ab30f23dc902d443633a3927974f89f6f06